src/static_libs/vg_common/vg_common_svg.c

```diff
@@ -540,6 +540,10 @@ vg_common_svg_node_free(Svg_Node *node)
            if (node->node.command.commands) free(node->node.command.commands);
            if (node->node.command.points) free(node->node.command.points);         
            break;
+        case SVG_NODE_CUSTOME_COMMAND:
+           if (node->node.command.commands) free(node->node.command.commands);
+           if (node->node.command.points) free(node->node.command.points);
+           break;
```

Comment:
This case seems to be the same as its previous one.

Reply:
Indeed.